Week-23/todo-list/Router/TodoRouter.js

```diff
@@ -0,0 +1,16 @@
+const {
+  postTodo,
+  getTodos,
+  getTodoByID,
+  updateTodo,
+  deleteById,
+} = require("../Controller/TodoController");
+
+const router = require("express").Router();
+
+router.route("/").post(postTodo);
+router.route("/").get(getTodos);
+router.route("/:id").get(getTodoByID);
+router.route("/:id").put(updateTodo);
+router.route("/:id").delete(deleteById);
```

Comment:
See if you can refactor these to avoid extra route(), something like router.get(“/“)